DuckDuckGo/Base.lproj/Settings.storyboard

```diff
@@ -31,7 +31,7 @@
                                                 <label opaque="NO" multipleTouchEnabled="YES" contentMode="left" text="Set as Default Browser" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" id="xof-5k-PkI">
                                                     <rect key="frame" x="20" y="0.0" width="334" height="43.666667938232422"/>
                                                     <autoresizingMask key="autoresizingMask"/>
-                                                    <fontDescription key="fontDescription" type="system" weight="regular" pointSize="16"/>
+                                                    <fontDescription key="fontDescription" type="system" pointSize="16"/>
```

Comment:
Are these changes intentional?

From the description of the PR I'd expect the NetP item would change to match the pre-existing fonts from other setting items.

Reply:
@graeme - Wow you're right, just saving the storyboard from develop with no changes did this for me too.  It seems something changed in the default storyboard specs that these changes are the new defaults.